commit b770266fa9c3fab5374bdf109676e0b129ac4546
Author: Bryce Harrington <bryce@canonical.com>
Date:   Wed Apr 3 13:36:20 2013 -0700

    Don't bother probing -nv on linux.
    
    For linux this driver is long obsolete now.  It may have some relevance
    on non-linux systems.

diff --git a/hw/xfree86/common/xf86pciBus.c b/hw/xfree86/common/xf86pciBus.c
index b2545be..73a662d 100644
--- a/hw/xfree86/common/xf86pciBus.c
+++ b/hw/xfree86/common/xf86pciBus.c
@@ -1220,8 +1220,9 @@ xf86VideoPtrToDriverList(struct pci_device *dev,
         driverList[idx++] = "nvidia";
 #ifdef __linux__
         driverList[idx++] = "nouveau";
-#endif
+#else
         driverList[idx++] = "nv";
+#endif
         break;
     }
     case 0x1106:
